tests/Php54/Php54Test.php

```diff
@@ -34,4 +34,16 @@ public function testClassUsesInvalid()
     {
         $this->assertFalse(@class_uses('NotDefined'));
     }
+
+    public function testHexToBinValid()
+    {
+        $this->assertEquals("\x61\x62\x00\x63\x64", hex2bin("6162006364")); // With null byte, see #48
```

Comment:
:-1: for referencing a PR in test.
If this case is somehow special it's enough to describe it here.